stl/inc/chrono

```diff
@@ -2215,6 +2215,36 @@ namespace chrono {
         return hours{_Ret};
     }
 
+    // [time.zone.timezone]
+
+    // TRANSITION: work in progress
+    class time_zone {
+    public:
+        time_zone(std::string_view _Name) : _Name(_Name) {}

```

Comment:
This internal constructor should be `explicit` (because implicitly converting constructors are hazardous except when the class is designed around them).